kuksa-persistence-provider/Readme.md

```diff
@@ -129,4 +136,5 @@ cargo build --features json_djson --no-default-features
         # Set Vales in cmd
         setValue Vehicle.Cabin.HVAC.Station.Row4.Passenger.FanSpeed 40
         ```
+
```

Comment:
When fixing the comment above also fix `Vales` a few lines above

Reply:
Done